src/ui.text-notify/config.ts

```diff
@@ -7,13 +7,23 @@ export default /*tw*/ {
     enterFromClass: "opacity-0",
     leaveToClass: "opacity-0",
   },
-  body: `
-    mb-3 flex w-full items-center justify-center gap-3 rounded-2xl bg-gray-900/90
-    p-4 shadow-[0_4px_16px_rgba(17,24,39,0.5)] backdrop-blur-md md:shadow-[0_0px_12px_rgba(0,0,0,0.25)]
-  `,
-  bodySuccess: "bg-[radial-gradient(100.16%_500.78%_at_0%_50%,rgba(74,222,128,0.1)_2.17%,transparent)]",
-  bodyWarning: "bg-[radial-gradient(100.16%_500.78%_at_0%_50%,rgba(251,146,60,0.2)_2.17%,transparent)]",
-  bodyError: "bg-[radial-gradient(100.16%_500.78%_at_0%_50%,rgba(251,113,133,0.2)_2.17%,transparent)]",
+  body: {
+    base: `
+      mb-3 flex w-full items-center justify-center gap-3 rounded-2xl bg-gray-900/90
+      p-4 shadow-[0_4px_16px_rgba(17,24,39,0.5)] backdrop-blur-md md:shadow-[0_0px_12px_rgba(0,0,0,0.25)]
+    `,
+    variants: {
+      bodySuccess: {
+        true: "bg-[radial-gradient(100.16%_500.78%_at_0%_50%,rgba(74,222,128,0.1)_2.17%,transparent)]",
+      },
```

Comment:
Make it simpler:
`variants.type.success = "bg-[radial-gradient(100.16%_500.78..."`